lib/application/pages/home/collections/update/update_collection_page.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/pages/home/collections/update/update_collection_details.dart';
+import 'package:memo/application/pages/home/collections/update/update_collection_memos.dart';
+import 'package:memo/application/pages/home/collections/update/update_providers.dart';
+import 'package:memo/application/theme/theme_controller.dart';
+import 'package:memo/application/view-models/home/update_collection_vm.dart';
+import 'package:memo/application/widgets/theme/custom_button.dart';
+import 'package:memo/application/widgets/theme/retry_container.dart';
+import 'package:memo/application/widgets/theme/themed_container.dart';
+import 'package:memo/application/widgets/theme/themed_tab_bar.dart';
+
+enum Segment { details, memos }
```

Comment:
too broad of a name

Reply:
shouldn't be private?

Reply:
> too broad of a name

Do you think? I mean, considering that it's private which means that it applies to this screen only, isn't clear that it references to the tab bar segment?

Reply:
Yeah, it will pollute the global namespace with "Segment"

Reply:
But it's private now. It's only visible inside the page